perfkitbenchmarker/benchmark_spec.py

```diff
@@ -298,7 +303,8 @@ def PrepareVm(self, vm, firewall):
     vm.Create()
     logging.info('VM: %s', vm.ip_address)
     logging.info('Waiting for boot completion.')
-    firewall.AllowPort(vm, vm.ssh_port)
+    if vm.ssh_port:
```

Comment:
If this is in preparation for windows support, maybe VMs should expose a set of ports required for remote access?